docker.go

```diff
@@ -85,6 +85,16 @@ func isPortSet(container types.ContainerJSON, svcType string, svcName string) bo
 	return false
 }
 
+// Check if the network name is explicitly set via label
+func isNetworkSet(container types.ContainerJSON) bool {
+	for k := range container.Config.Labels {
```

Comment:
Judging from the change at line 222 below, I think this method could be simplified down to `return container.Config.Labels["traefik.docker.network"] != ""`